snippets/rust/src/webhook.rs

```diff
@@ -0,0 +1,21 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use breez_sdk_core::InputType::LnUrlWithdraw;
```

Comment:
This line is not needed. Maybe add `webhook` to `main.rs` so the CI picks it up.